docker-compose.test.yaml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
+
+services:
+  postgresql:
+    command: "postgres -c 'shared_buffers=128MB' -c 'fsync=off' -c 'synchronous_commit=off' -c 'full_page_writes=off' -c 'max_connections=100' -c 'client_min_messages=warning'"
+    env_file:
+      - .env.test
+
+  celery-worker:
```

Comment:
We won't use the celery-worker in the test cases because fixtures by celery will replace it. That means we'll never start the celery-worker container when we want to run tests. Do you know if we still need it here?